### ISSUE TITLE
g.IdleTime() Example fails

### ISSUE THREAD

**Issue by SegundoBob** [leo-editor/leo-editor#487]
See [Running Code at Idle time](http://leoeditor.com/scripting-miscellany.html#running-code-at-idle-time ) for the g.IdleTime() example.

When I put this example in a Leo-Editor node and run it by pressing Ctrl-B, I get the following errors:
```
exception executing script
Traceback (most recent call last):
  File "/pri/git/leo-editor/leo/core/leoCommands.py", line 944, in executeScript
    c.executeScriptHelper(args, define_g, define_name, namespace, script)
  File "/pri/git/leo-editor/leo/core/leoCommands.py", line 991, in executeScriptHelper
    exec(compile(script, scriptFile, 'exec'), d)
  File "/sec/tmp/leoScript.py", line 18, in <module>
    it1 = g.IdleTime(c,handler1,delay=500)
TypeError: IdleTime() got multiple values for argument 'delay'
--------------------
  line 16:         it.stop()
* line 17: 
  line 18: it1 = g.IdleTime(c,handler1,delay=500)
  line 19: it2 = g.IdleTime(c,handler2,delay=1000)
```
Test System:
```
Leo Log Window
Leo 5.5, build 20170428101630, Fri Apr 28 10:16:30 CDT 2017
Git repo info: branch = master, commit = f9308f0acacf
Python 3.5.2, PyQt version 5.5.1
linux
```

**Comment by edreamleo** [comment]
Thanks for this. The c argument no longer exists.  I'll fix the example in the docstring.

**Comment by SegundoBob** [comment]
Thanks.

Dropping the commander from the call, I get this error:
```

Traceback (most recent call last):
  File "/pri/git/leo-editor/leo/plugins/qt_idle_time.py", line 115, in call_handler
    self.handler(self)
  File "/sec/tmp/leoScript.py", line 6, in handler1
    g.trace('%2s %s %2.4f' % (it.count,it.c.shortFileName(),delta_t))
AttributeError: 'IdleTime' object has no attribute 'c'
Traceback (most recent call last):
  File "/pri/git/leo-editor/leo/plugins/qt_idle_time.py", line 115, in call_handler
    self.handler(self)
  File "/sec/tmp/leoScript.py", line 13, in handler2
    g.trace('%2s %s %2.4f' % (it.count,it.c.shortFileName(),delta_t))
AttributeError: 'IdleTime' object has no attribute 'c'
```

**Comment by edreamleo** [comment]
On Fri, Apr 28, 2017 at 3:36 PM, SegundoBob <notifications@github.com>
wrote:

> Dropping the commander from the call, I get this error:
>
​Fixed (and tested) at 44fa6f7.

Edward
